### PR TITLE
Droth 15 vehicle limitations and turning limitations

### PR DESCRIPTION
**Pull request by leirasf**
Vehicle limitations and turning limitations: Also minutes every 5 minutes to period of validity

### CODE REVIEW

File: digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/LinearAsset.scala [hunk 1]
```diff
@@ -26,29 +26,67 @@ case class Prohibitions(prohibitions: Seq[ProhibitionValue]) extends Value {
...
+  def endHour: Int
+  def days: ValidityPeriodDayOfWeek
+  def duration(): Int
+  def preciseDuration(): (Int, Int)
+  def startMinute: Int
+  def endMinute: Int
+}
+
+case class ValidityPeriod(override val startHour: Int, override val endHour: Int, override val days: ValidityPeriodDayOfWeek,
+                          override val startMinute: Int = 0, override val endMinute: Int = 0) extends ValidityPeriodsData{
```
Comment (leirasf) [inline digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/LinearAsset.scala]: I think we don't need to have the trait "ValidityPeriodsData"

**Comment by halied** [comment]
Changes have been added to VAK (hazardous material) limitations too, which was not mentioned in user story, but I suppose it's ok.

**Comment by leirasf** [comment]
Looks good to me now

**Comment by marquesfm** [comment]
Looks good to me.

**Pull request merged**